modules/tinymce/src/themes/silver/main/ts/api/Options.ts

```diff
@@ -348,6 +348,8 @@ const getSkinUrl = (editor: Editor): string | undefined => {
   }
 };
 
+const getSkinUrlOpt = (editor: Editor): Optional<string> => Optional.from(editor.options.get('skin_url'));
```

Comment:
```suggestion
const getSkinUrlOpt = wrapOptional(option('skin_url'));
```

Also, following other uses in this file I suggest renaming it to `getSkinUrlOption` but that would break your code if I made it a suggestion.

Reply:
Done!